frontend/haldirams-explorer/haldirams.css

```
@@ -0,0 +1,528 @@
+/* =========================================
+   HALDIRAM'S PAGE
+   ========================================= */
+
+.haldirams-main {
+    background:
+        radial-gradient(circle at 15% 20%, rgba(180, 83, 9, 0.12), transparent 30%),
+        radial-gradient(circle at 85% 35%, rgba(120, 53, 15, 0.12), transparent 30%),
+        #09090b;
+    color: #fafafa;
+    font-family: "Outfit", sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)